src/molecules/top-navigation-bar.tsx

```diff
@@ -12,15 +16,25 @@ const UserAvatar = ({ userInitial }: { userInitial?: string }) => (
 export type TopNavBarProps = {
   userInitial?: string;
   label?: string;
+  openSideMenu: () => void;
```

Comment:
```suggestion
  openSideMenu: () => void;
  subjects: SideMenuProps["subjects"];
```
型定義が変化しているようですので, このように修正している必要があるでしょう. (以下, 残りの変更の提案が別々のコメントに分かれています)